docs/configuration/production.rst

```diff
@@ -378,3 +378,55 @@ The sample *Nginx* configuration file shown above will look like so:
             include     /path/to/uwsgi_params; # the uwsgi_params file previously downloaded
         }
     }
+
+
+Upgrading Kinto
+===============
+
+.. important::
+
+    We follow `semver <http://semver.org/>`_ for version numbers.
+
+    Before upgrading, read the release notes about potential breaking changes.
+
+    See also ref:`API versioning <api-versioning>`.
+
+First, make the potential changes to the configuration file, as described in
+the release notes.
+
+If installed as Python package, make sure the virtualenv is activated:
+
+::
+
+    source env/bin/activate
+
+Now upgrade Kinto (and its dependencies) using the following command:
+
+::
+
+    pip install --upgrade kinto
+
+Since expected database schemas may change, do not forget to run the migration with:
```

Comment:
The beginning of this sentence is weird, did you mean something like "Since there might be some database schema changes, ..." ?